### PR TITLE
 Internationalization (German) of the mobile app.

### PR DESCRIPTION
**Pull request by matthinc**

### CODE REVIEW

**Review by alextran1502 — COMMENTED** [review]
Thank you for a wonderful PR. Can you help me add a subsection in the section `Development` in the README of how to set Localizely up and running?

File: mobile/lib/modules/asset_viewer/ui/exif_bottom_sheet.dart [hunk 1]
```diff
@@ -93,12 +94,12 @@ class ExifBottomSheet extends ConsumerWidget {
           Padding(
             padding: const EdgeInsets.only(top: 16.0),
             child: Text(
-              "Add Description...",
+              "asset_description",
```
[the remark this answers is not in the document]
Comment (matthinc) [reply mobile/lib/modules/asset_viewer/ui/exif_bottom_sheet.dart]: Good idea! I changed the keys, except those (few) which are used across multiple dart files.

File: localizely.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+config_version: 1.0
+project_id: ead34689-ec52-41d9-b675-09bc85a6cbd7
+file_type: flutter_arb
+upload:
+  files:
+    - file: mobile/assets/i18n/en-US.json
+      locale_code: en
+    - file: mobile/assets/i18n/de-DE.json
```
Comment (zackpollard) [inline localizely.yml:8]: Should this be in the upload section? I believe we should have just en-US in the upload section, then have all the other languages in the download section so localizely updates them for us automatically?

Comment (matthinc) [reply localizely.yml:8]: Personally I would add all existing JSON files to the upload section. That way manual JSON changes and localizely are always in-sync. Localizely seems to do a great job merging local and remote changes.

**Pull request merged**